community/SDK-GOVERNANCE.md

```diff
@@ -10,7 +10,42 @@ The community is organized as follows:
 
 ## New maintainers
 
-TBD
+Maintainers are a crucial part of this community. They keep our projects alive
+and ensure both development and maintenance. We define a set of criteria,
+followed by a vote, to include an active contributor as maintainer of one or
+more SDK projects. An active contributor may propose themselves as a maintainer,
+although we encourage the existing maintainers to nominate active contributors
+proactively.
+
+The criteria for a contributor to become a maintainer for a given SDK repository
+are:
+
+1. At least 20 reviews on eligible PRs
+1. At least 20 submitted eligible PRs as author
```

Comment:
I would make it clear that this is an AND. e.g.
```
1. At least 20 reviews on eligible PRs, _AND_
1. At least 20 submitted eligible PRs as author
```

Reply:
+1 to the "AND".  Based on last week's call perhaps something like:
```
1. Significant number, and regular, reviewer on non-trivial PRs, _AND_
1. Significant number, and regular, author of non-trivial PRs 

Note that the definition of "significant" is purposely left as undefined since it is
very subjective, and the purpose of these requirements are to demonstrate the
person's expertise and commitment to the project - not simply to achieve a
certain level of activity
```

Maybe?